### PR TITLE
Fix orders controller spec

### LINKED ISSUE #2685: [Spree Upgrade][Models] Fix order_spec and orders_controller spec tests with shipment fees and taxes (3 broken tests)
Originated from #2009.
We have to find what's broken in the shipping fees calculation and fix them in spec/models/spree/order_spec.rb. It may be a code change, it may just be adapting tests.

Acc Criteria: order_spec tests passing.

### PR DESCRIPTION
**Pull request by sauloperez**
#### What? Why?

Closes #2685

`spec/controllers/spree/orders_controller_spec.rb:227` was not passing. This fixes it.

#### What should we test?
All spec/controllers/spree/orders_controller_spec.rb specs should pass

### CODE REVIEW

**Comment by sauloperez** [comment]
Passing :green_apple:

File: spec/controllers/spree/orders_controller_spec.rb [hunk 1]
```diff
@@ -224,7 +224,7 @@
           } }
         }
 
-        expect(order.adjustments.map(&:state)).to eq(['closed', 'closed', 'closed'])
+        expect(order.adjustments.map(&:state)).to eq(['closed', 'open', 'closed'])
```
[the remark this answers is not in the document]
Comment (sauloperez) [reply spec/controllers/spree/orders_controller_spec.rb]: See my last commit. I hope it helps. What I did find out was that we used a custom event to implement our own enterprise fees. I haven't explored in detail how all that works though. There's more stuff about adjustments than we think :scream:

File: spec/controllers/spree/orders_controller_spec.rb [hunk 2]
```diff
@@ -224,7 +224,16 @@
           } }
         }
 
-        expect(order.adjustments.map(&:state)).to eq(['closed', 'closed', 'closed'])
+        # Before issuing the update the secod adjustment, which is associated
```
Comment (luisramos0) [inline spec/controllers/spree/orders_controller_spec.rb]:
```suggestion
        # Before issuing the update, the second adjustment, which is associated
```

File: spec/controllers/spree/orders_controller_spec.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (luisramos0) [reply spec/controllers/spree/orders_controller_spec.rb]: yeah, the way it's triggered is odd but then there are the enterprise fees adjustments that plug into the adjustments party as the other adjustments with calculators and taxes and brain damaging logic 🤣

[the remark this answers is not in the document]
Comment (luisramos0) [reply spec/controllers/spree/orders_controller_spec.rb]: thanks for the comment, looks much better. there's just a typo.

**Comment by sauloperez** [comment]
The test is passing :tada:

**Pull request merged**